pkg/diagnostics/network/objects.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	diagnosticsImage           = "openshift/diagnostics-deployer"
+	diagnosticsImage           = "openshift/origin"
```

Comment:
I don't see existing image format string for 'openshift/origin', there is openshift/origin-${component}:${version} for system components but that doesn't match with what we need. 
Supporting this as config involves creating Network diags section with an option in master-config 
and this needs to be propagated all the way to GetNetworkDiagnosticsPod(). 
Is it easy to patch OCP instead?